engine/include/cubos/engine/voxels/grid.hpp

```diff
@@ -8,22 +8,16 @@
 
 #include <glm/glm.hpp>
 
-#include <cubos/core/data/old/deserializer.hpp>
-#include <cubos/core/data/old/serializer.hpp>
 #include <cubos/core/reflection/reflect.hpp>
 
+#include <cubos/engine/assets/bridges/file.hpp>
```

Comment:
You don't need this anymore!
Since you use `cubos::core::memory::Stream&` down there, you can forward declare it instead of including it. Saves compiling time.

```cpp
namespace cubos::core::memory
{
    class Stream;
} // namespace cubos::core::memory